### PR TITLE
Lang PT-BR

### PR DESCRIPTION
**Pull request by maisondasilva**

### CODE REVIEW

**Review by XFox111 — COMMENTED** [review]
Hi, thanks for your contribution!
Your PR seems good. I'll include it into next Patch Tuesday update

Please update PR description, so it contains summary of the proposed changes

Also, please send me Brazilian Portuguese translation for the following text. It will go into description section on the store page

```text
Extension for web browsers which helps you to easily generate strong passwords in one click

# Features
- Create strong passwords in one click
- Customizable generator
- Clean and simple UI
- Dark mode

# Hey, It's open-source software!
If you know how to improve this extension you can check its GitHub Repository
Check out release changelog
```

---
One more thing. As described in [Contribution Guidelines](https://github.com/XFox111/PasswordGeneratorExtension/blob/main/CONTRIBUTING.md#contributing-to-translations), next time you want to start working on a pull request, make sure you've been assigned for an issue:

> Note that whatever you want to contribute to the codebase, you should do it only after you got assigned on an issue

It is vital to show that you are working on the issue, so other people don't start working on it too

**Comment by maisondasilva** [comment]
```
Extensão para navegadores da web que ajuda você a gerar facilmente senhas fortes com um clique

# Características
- Crie senhas fortes com um clique
- Gerador personalizável
- UI limpa e simples
- Modo escuro

# Ei, é um software de código aberto!
Se você sabe como melhorar esta extensão, você pode verificar seu repositório GitHub
Confira o changelog da versão
```

**Review by XFox111 — DISMISSED** [review]

**Comment by maisondasilva** [comment]
Change pt-br for pt_BR

**Comment by XFox111** [comment]
> Change pt-br for pt_BR

Oh, yeah. It indeed should be `pt_BR`. Guess, I need to make better reviews 😅

**Comment by XFox111** [comment]
@maisondasilva yeah, sure. I'll just come back to it a bit later

File: public/_locales/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+	"Copy": {
+		"message": "Copiar",
+		"description": "PasswordView.tsx"
+	},
+	"Generate_new": {
+		"message": "Gerar nova",
+		"description": "PasswordView.tsx"
+	},
+	"Exclude_special_symbols_one_time": {
+		"message": "Generate password without special symbols",
```
Comment (XFox111) [inline public/_locales/pt_BR/messages.json:27]: I think, it should be translated as well

**Review by XFox111 — CHANGES_REQUESTED** [review]
Yep. I had to be tested first. I've found a line that you missed :)

**Review by XFox111 — APPROVED** [review]
Ok, I think it's good to go!
Thanks for your contribution :)

**Comment by XFox111** [comment]
Jeez. Looks like I need a break. Sorry, I messed up usernames and put another person. Fixed it now 😅

**Comment by maisondasilva** [comment]
I asked because I didn't see him in this translation, no problem and good rest 😂😂😅😅

**Comment by maisondasilva** [comment]
@XFox111 Repeating is Is there an option to exclude repeated passwords, that is, passwords that have already been generated and prevent it from being generated again, correct?

image

**Comment by XFox111** [comment]
> @XFox111 "Repeating" is Is there an option to exclude repeated passwords, that is, passwords that have already been generated and prevent it from being generated again, correct?

I'm not sure if I understand your question correctly. The "Repeating" option goes under "Character options -> Exclude" section. So, it prevents characters that already has been added to the password from being added again (every character in generated password becomes unique)

**Pull request merged**